presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -25,6 +25,29 @@ data warehouse. Hive is a combination of three components:
 Presto only uses the first two components: the data and the metadata.
 It does not use HiveQL or any part of Hive's execution environment.
 
+Requirements
+------------
+
+The Hive connector requires a Hive metastore service (HMS). The following
+versions are supported:
+
+* Apache Hive 1.x
+* Apache Hive 2.x
+
+The following Hadoop distributions, and the included HMS, are supported.
+
+* Apache Hadoop 2.x
+* Apache Hadoop 3.x
+
+Derivative distributions using the above version, including Cloudera CDH 5 and
+Hortonworks Data Platform (HDP) are supported as well.
+
+Alternatively, you can use compatible implementations of a Hive metastore, such
+as `AWS Glue <https://aws.amazon.com/glue/>`_.
+
+Many other distributed object stores are supported, as long as the metadata is
```

Comment:
We could/should probably add more info about this. But not sure what we want to have in the long list specifically.

Reply:
related to https://github.com/prestosql/presto/issues/3471.  supported object stores: gcs, s3, wasb, abfs, adl, hdfs. not sure about others like aliyun/ibm, or s3 'compatible ones' like minio

Reply:
I know ... which ones of these should we add to the docs though .. and which ones fall under. .. "and others segment".

Reply:
We can list a few here. I'll update it.